common/config/config.go

```diff
@@ -254,3 +255,24 @@ func ShouldVerifyVersion() bool {
 
 	return true
 }
+
+func DockerRepoOverride(image string) string {
+	v := get(DOCKER_REPO_OVERRIDE)
+	overrideMap := map[string]string{}
+	for _, s := range strings.Split(v, ",") {
+		if mapping := strings.Split(s, ":"); len(mapping) == 2 {
+			overrideMap[mapping[0]] = mapping[1]
+		}
+	}
+
+	imageParts := strings.Split(image, "/")
+	if len(imageParts) < 2 {
+		return image
```

Comment:
[Nit] We could fail even earlier and avoid a config lookup by moving this block up to the top

Reply:
Do you mean the image parts check? I agree. It isn't a case I would expect to fail normally though. More of a mandatory check everytime we use split.